functests/setup_teardown.go

```diff
@@ -17,7 +18,7 @@ func BeforeTestSuiteSetup() {
 	err = t.CreateNamespace(TestNamespace)
 	gomega.Expect(err).To(gomega.BeNil())
 
-	err = t.DeployOCSWithOLM(ocsRegistryImage, localStorageRegistryImage)
+	err = t.DeployOCSWithOLM(OcsRegistryImage, LocalStorageRegistryImage, OcsSubscriptionChannel)
```

Comment:
> I structured the upgrade process to go into the setup method so that we can run the same set of test specs on top of a fresh deployment or an upgraded environment.

I get your thought process here. I'm unsure if this lets us express some test cases though.

Take this test case for example (which is something I'd like to eventually see)

- Install OCS, create Storage Cluster
- create rbd block pvc and attach it to a pod that continually performs reads/writes to pvc, pod fails if read/writes fail
- update OCS and StorageCluster, wait for update to complete
- ensure workload pod wasn't interrupted

If we perform the update at setup, then we can't validate potential workload impact that might occur during the update.

For now, can we have a functional test that updates OCS, and the transitions back to the tested version of OCS?

Reply:
> transitions back to the tested version of OCS
What's the best way to do that? OLM doesn't support rollbacks/downgrades. So uninstall and reinstall?

Reply:
Well, we need uninstall test logic anyway. So there's value in investing in that now as well.

I'd say go for uninstall/reinstall.

Reply:
@davidvossel Updated with the changes requested. PTAL